### PR TITLE
Updated database dependencies and added custom message to parseOrFail

### PR DESCRIPTION
**Pull request by ProteinDev**
Please accept this, it'd be great.

### CODE REVIEW

File: helper/pom.xml [hunk 1]
```diff
@@ -89,6 +89,7 @@
                                     <include>org.spongepowered:configurate-hocon</include>
                                     <include>com.typesafe:config</include>
                                     <include>net.kyori:event-api</include>
+                                    <inclue>org.slf4j:*</inclue>
```
Comment (md678685) [inline helper/pom.xml]: > `inclue`

Comment (ProteinDev) [reply helper/pom.xml]: the latest version of Jedis doesn't package slf4j

Comment (md678685) [reply helper/pom.xml]: Should be `include` not `inclue`.

**Comment by lucko** [comment]
The issues raised here are addressed in a295db5ee1926aa78165484875babe9d0de9b38a and 66cad4010942c21d6a360057673401ce9440952a. 

The dependency versions are now in-line with what I use in LuckPerms - I know those versions are relatively stable. I'm particularly skeptical about updating to the new major Jedis release.

Aand, I wasn't too keen on the approach to allowing custom parseOrFail messages in this PR, so I've adapted it slightly and implemented something similar in the first commit linked above.

Sorry to not merge your PR, but I thought it'd be much easier if I just made the changes myself than to ask you to make lots of changes. :) Hope you understand.

**Pull request closed without merging**